src/api/twitter.js

```diff
@@ -0,0 +1,41 @@
+class TwitterHandler {
+    constructor (twitterMgr, claimMgr){
+        this.twitterMgr = twitterMgr
+        this.claimMgr = claimMgr
+    }
+
+    async handle(event, context, cb){
+        let body
+        try {
+            body = JSON.parse(event.body)
+        } catch (e) {
+            cb({ code: 400, message: 'no json body: ' + e.toString() })
+            return
+        }
+
+        if (event.headers.Origin != "Origin: https://3box.io") {
```

Comment:
We need to be able to handle any 3box.io subdomain as well. Right now we are testing the dapp on develop.3box.io for example

Reply:
Good catch. I'll fix it